cvat/settings/base.py

```diff
@@ -56,8 +56,8 @@ def add_ssh_keys():
         IGNORE_FILES = ('README.md', 'ssh.pid')
         keys_to_add = [entry.name for entry in os.scandir(ssh_dir) if entry.name not in IGNORE_FILES]
         keys_to_add = ' '.join(os.path.join(ssh_dir, f) for f in keys_to_add)
-        subprocess.run(['ssh-add {}'.format(keys_to_add)],
-            shell = True,
+        subprocess.run(['ssh-add {}'.format(keys_to_add)], # nosec
```

Comment:
@Marishka17 , I hope you checked the functionality and it still works. I agree that shell=True should be removed. Just need to be sure that ssh keys work as expected.